Resources/Textures/Nyanotrasen/Clothing/Mask/felinidcone.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "katayusha",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "icon"
+      },
+      {
+        "name": "inhand-left",
+        "directions": 4
+      },
+      {
+        "name": "inhand-right",
```

Comment:
just drop the inhand states if there's no sprite for it. delete the 2 inhand pngs and delete the states here

Reply:
there are inhand sprites

Reply:
my bad